register-html-template.js

```diff
@@ -39,10 +39,10 @@ class RegisterHtmlTemplate {
       div.innerHTML = trimmedVal;
       if (div.firstChild) {
         if (document.body) {
-          document.body.insertBefore(div.firstChild, document.body.firstChild);
+          document.body.appendChild(div.firstChild, document.body.firstChild);
         } else {
           document.addEventListener('DOMContentLoaded', () => {
-            document.body.insertBefore(div.firstChild, document.body.firstChild);
+            document.body.appendChild(div.firstChild, document.body.firstChild);
```

Comment:
If order matters, this is a bit interesting. I _think_ it should be ok because event listeners fire in the order they are added.

Again though `appendChild` takes a single argument.

Reply:
Done